### ISSUE TITLE
How to check for over-training?

### ISSUE THREAD

**Issue by maxsenh** [facebookresearch/maskrcnn-benchmark#707]
## ❓ Questions and Help
Hi and thanks again for this beautiful implementation!

My question is simple, how to check for over-training?
As far as I understand, over-fitting happens when using a high number of epochs/iterations and a small data set and the model starts to mesmerize instead of learn the data. Nevertheless, that might be checked by comparing the training loss and the validation loss, but the latter is not implemented right?
Is there another way of checking for over-fitting? How did one check for over-training when running the COCO data set?

Thanks in advance for the help!
